docs/content/latest/admin/yb-admin.md

```diff
@@ -106,7 +106,7 @@ If you need to take a node down temporarily, but intend to bring it back up, you
 * If the node is down for less than 15 minutes, it will catch up through RPC calls when it comes back online.
```

Comment:
The list just above has ADD SERVER and REMOVE SERVER in all these

Reply:
Good catch! Fixed in the latest commit.